src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { Theme, styled } from '@mui/material/styles';
+import MuiStepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepLabel from '@mui/material/StepLabel';
+import NavigateNext from '@mui/icons-material/NavigateNext';
+import { StepIconProps } from '@mui/material/StepIcon';
+
+const CustomStep = styled(Step)(({ theme }) => ({
+  padding: theme.spacing(1),
+}));
+
+const stepStateStyles = (theme: Theme) => ({
+  ...theme.typography.headline_ss_xxs,
+  color: `${theme.palette.grey['500']}`,
+  '& .MuiStepLabel-label.Mui-active': {
+    color: theme.palette.secondary.main,
+  },
+  '& .MuiStepLabel-label.Mui-completed': {
+    color: theme.palette.primary.light,
```

Comment:
not sure if it will be the case but having colors already defined here might be a problem if we want a stepper in a different color no?

Reply:
yeah, but the stepper does not have color variant
Only default and error states